tests/test_room_wave_recovery.py

```
@@ -0,0 +1,363 @@
+"""‏#217 — הסבב אינו נשאר תלוי כשהגל שלו נעלם מתחתיו.
+
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)